var/spack/repos/builtin/packages/py-sphinx-design/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxDesign(PythonPackage):
+    """A sphinx extension for designing beautiful, screen-size responsive web components."""
+
+    homepage = "https://sphinx-design.readthedocs.io"
+    url = "https://github.com/executablebooks/sphinx-design/archive/refs/tags/v0.3.0.tar.gz"
+    git = "https://github.com/executablebooks/sphinx-design.git"
```

Comment:
```suggestion
    git = "https://github.com/executablebooks/sphinx-design.git"
    list_url = "https://pypi.org/simple/sphinx-design/"
```

Reply:
This isn't needed if you use the `pypi` attribute instead of `url`.